src/search/search-index-new/add.js

```diff
@@ -27,13 +27,19 @@ import pipeline from './pipeline'
  * @return {Promise<void>}
  */
 export async function addPage({ visits = [], bookmark, ...pipelineReq }) {
-    const pageData = await pipeline(pipelineReq)
-    const timerLabel = `TIMER - add page: "${pageData.url}"`
+    const { favIconURI, ...pageData } = await pipeline(pipelineReq)
 
-    console.time(timerLabel)
     try {
         await db.transaction('rw', db.tables, async () => {
             const page = new Page(pageData)
+
+            // Record a new fav-icon, if present (continue on straight-away)
```

Comment:
Head to read over this comment twice to understand, reading the code along with it  ;)  Then, this comment basically says the same as the code, which reads pretty clearly, so maybe the comment is redundant? Usually I try to make the code itself speak as human as possible by having good variable and function names, keeping expressions as simple as possible, while commenting things that are not immediately obvious. (Like 'store this version of the favicon first, so we can do this and this more efficiently later'.) Maybe something for a style guide that we can discuss and make explicit?

Reply:
Yeah I'm pretty good at redundant comments 😄 also good at forgetting about comments, then changing the code so that comments that make no sense anymore. Too bad we can't lint English. I like you're idea of trying to focus on not needing to put comments there in the first place though. I will play around with this.